admiral/crud/interfaces.ts

```diff
@@ -25,6 +25,26 @@ export type CRUDActionsLocale = {
     paginationTotal: (total: number) => string
 }
 
+export type CRUDFormCreate =
+    | {
+          fields: React.ReactNode
+          children?: never
+      }
+    | {
+          fields?: never
+          children: React.ReactNode
+      }
+
+export type CRUDFormEdit =
+    | {
+          fields: React.ReactNode
+          children?: never
+      }
+    | {
+          fields?: never
+          children: React.ReactNode
+      }
+
```

Comment:
1. You can combine two types into one and name it - "CRUDForm".
2. Why did you use the "never" type if JSX elements are always passed here?